site/app/templates/admin/Extensions.twig

```diff
@@ -54,6 +54,7 @@
                     </select>
                 </span>
             </label>
+            <p><strong>Note:</strong> The optional reason for extension text will be displayed to the student on the Rainbow Grades page</p>
```

Comment:
```suggestion
            <p><strong>Note:</strong> The optional reason for extension text will be displayed to the student on the Rainbow Grades page.</p>
```

Reply:
I have fixed this issue. Also, the reason why you felt that way is because of the strong text. I removed it and checked whether it is okay or not.

Reply:
I think you should keep the `strong` and the spacing makes it a bit more clear the the text is for the row not the button in my opinion. Also make sure to add the period.